Fibo1.js

```diff
@@ -2,12 +2,7 @@
  * Created by User on 3/17/14.
  */
 function fibonaci(n) {
-    if ((1 == n) || (0 == n) )  {
-        return 1;
-    }
-    else {
-        return (fibonaci(n-1) + fibonaci(n-2));
-    }
+    return (Math.pow((1+Math.sqrt(5))/2,n)-Math.pow((1-Math.sqrt(5))/2,n))/Math.sqrt(5);
 }
 
 console.log('num is', fibonaci(20));
```

Comment:
Please log properly using `loacker`

Reply:
Please keep the PR's with minimal changes, if you think we should change this log please create a separate PR for it
@moLow